### PR TITLE
Added missing Pt support to camera

### PR DESCRIPTION
**Pull request by ssscassio**
I noticed that pt support for camera was missing. Just added it.

### CODE REVIEW

File: app/assets/javascripts/uploadcare/locale/en.js.coffee [hunk 1]
```diff
@@ -1,7 +1,3 @@
-##
-## English locale is the default and used as a fallback.
-##
-
```
Comment (Zmoki) [inline app/assets/javascripts/uploadcare/locale/en.js.coffee]: Please, revert this deleting. It's needed comment.

File: app/assets/javascripts/uploadcare/locale/pt.js.coffee [hunk 2]
```diff
@@ -5,7 +5,6 @@
 ##
 ## Any fixes are welcome.
 ##
-
```
Comment (Zmoki) [inline app/assets/javascripts/uploadcare/locale/pt.js.coffee]: Revert newline

File: app/assets/javascripts/uploadcare/locale/pt.js.coffee [hunk 3]
```diff
@@ -100,4 +112,4 @@ uploadcare.namespace 'locale.translations', (ns) ->
 uploadcare.namespace 'locale.pluralize', (ns) ->
   ns.pt = (n) ->
     return 'one' if n == 1
-    'other'
+    'other'
```
Comment (Zmoki) [inline app/assets/javascripts/uploadcare/locale/pt.js.coffee]: Revert this too =))

**Comment by dmitry-mukhin** [comment]
@Zmoki

**Comment by ssscassio** [comment]
@Zmoki Can you check this now?

**Comment by Zmoki** [comment]
@ssscassio sorry for long reaction. Thank you!

**Pull request merged**